docs/app/pages/components/sections/select/select/snippets/app.ts

```diff
@@ -49,24 +49,25 @@ export class AppComponent implements OnInit {
+        this.dataSets.strings = ['United States', 'United Kingdom', 'Afghanistan' /*...*/];
```

Comment:
Confused about this as well. You've removed all but 3 data items from the plunker example.

Reply:
The flat select plunker example only has 3 data items in it (for whatever reason that may be).  I thought that since it is the newest version that there may have been some discussion on this and I updated the rest to keep them consistent.  I'm guessing now that the full set is preferred - sorry I shouldn't have assumed this.

Reply:
Alastair has just explained that having three items in the dataset is only for the layout in the snippets.  I'll updated all plunker examples with the full dataset.  Thanks